extensions/pwpush/src/push-password.tsx

```diff
@@ -0,0 +1,67 @@
+import { showToast, Toast, Clipboard, Form, ActionPanel, Action } from "@raycast/api";
+import { got } from "got";
+
+interface PushResponse {
+  url_token: string;
+  payload: string;
+  expire_after_days: number;
+  expire_after_views: number;
+  expired: boolean;
+  created_at: string;
+  updated_at: string;
+}
+
+async function createPush(payload: string, expireViews: number, expireDays: number): Promise<PushResponse> {
+  const response = await got.post("https://pwpush.com/p.json", {
+    form: {
+      "password[payload]": payload,
+      "password[expire_after_days]": expireDays.toString(),
+      "password[expire_after_views]": expireViews.toString(),
+    },
+    responseType: "json",
+  });
+
+  const data: PushResponse = response.body;
+  const url = `https://pwpush.com/p/${data.url_token}`;
+  await Clipboard.copy(url);
+  await showToast({ title: "Secret URL Copied!", message: url, style: Toast.Style.Success });
+  return data;
+}
+
+export default function Command() {
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm
+            title="Push Password"
+            onSubmit={async (values) => {
+              await createPush(values.secret, parseInt(values.expireViews), parseInt(values.expireDays));
+            }}
+          />
+        </ActionPanel>
+      }
+    >
+      <Form.TextArea id="secret" title="Secret" placeholder="Enter sensitive data to securely share…" />
+      <Form.Dropdown defaultValue="5" id="expireViews" title="Expire After Views" storeValue>
```

Comment:
The "expire after views" and "expire after days" can be modified to use `Form.TextArea`, allowing users to enter numbers. The restriction for "expire after views" is between 1 and 100, while the maximum for "expire after days" is 90 days.